src/docs/content/observability/observability-example/usecase.md

```diff
@@ -72,7 +76,6 @@ We performed the following actions:
 - Setup service connector between OCI logging and OCI monitoring with a new custom monitoring namespace.
 - Simulated the payment failures on MuShop application. 
 
-The orders service logs the payment failure details and the agent config filters the specific message that we setup as part of OCI logging agent setup (fluentd regexp parser).
-
-Service connector helps to send the payment failures onto OCI Monitoring which then helped us to view the metrics on a dashboard. We then set alarm for that metric to get alerts on our email. 
+The MuShop orders service logs the payment failure details and the `OCI Console -> agent config` helped to send all the pod logs to OCI logging.
```

Comment:
Maybe show the full navigation path and use exact spelling/casing of the services